### PR TITLE
Chore/profile picture fallback

### PR DESCRIPTION
**Pull request by okwast**
Depends on #48 
Depends on #71 

Adding the fallback image for profile pictures and the user avatar in the header.

### CODE REVIEW

File: frontend/components/Header.tsx [hunk 1]
```diff
@@ -49,7 +52,7 @@ export const Header: FC = () => {
       {isLoggedIn ? (
         <div className="flex bg-blue-400">
           <div className="cursor-pointer" onClick={openMenu}>
-            <Avatar imageUrl="" />
+            <Avatar imageUrl={user?.Image || "/images/common/mystery.svg"} />
```
Comment (fschindler) [inline frontend/components/Header.tsx]: Using the following would ensure the existence of the file (gives a TS error if the file doesn't exists):
```
import mysteryImg from '../public/images/common/mystery.svg`
...
<Avatar imageUrl={user?.Image || mysteryImg} />
```

**Review by fschindler — APPROVED** [review]
I guess this is also based on #71 ?!

Looks quite good, just one small improvement idea.

File: frontend/components/Header.tsx (hunk 1 above)
Comment (okwast) [reply frontend/components/Header.tsx]: I added next-images now and converted all images to being imported.

Nice hint 👍

Comment (fschindler) [reply frontend/components/Header.tsx]: I really like this improvement!

**Pull request merged**